packages/cardano-services-client/src/HandleProvider/KoraLabsHandleProvider.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  Cardano,
+  HandleProvider,
+  HandleResolution,
+  HealthCheckResponse,
+  NetworkInfoProvider,
+  ProviderError,
+  ProviderFailure,
+  ResolveHandlesArgs
+} from '@cardano-sdk/core';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This eslint disable is curious. What's the reason for it?

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/732#discussion_r1206350277 
For this to happen, I can only disable the error.